landbosse/excelio/XlsxReader.py

```diff
@@ -406,6 +406,13 @@ def create_master_input_dictionary(self, project_data_dataframes, project_parame
 
         incomplete_input_dict['cable_specs_pd'] = project_data_dataframes['cable_specs']
 
+        # Place development cost in the project list. However, in order to maintain
+        # backward compatibility fill this with 0 if it is not specified
+        if 'Development labor cost USD' in project_parameters:
+            incomplete_input_dict['development_labor_cost_usd'] = project_parameters['Development labor cost USD']
+        else:
+            incomplete_input_dict['development_labor_cost_usd'] = 0
+
```

Comment:
This does not appear to ensure backwards compatibility. It sets the value to zero rather than using the value that is input by the user in the project data file.

Reply:
This has been fixed--it now works with old validation input files.